prio/config.c

```diff
@@ -67,6 +73,15 @@ PrioConfig_new(int n_fields, PublicKey server_a, PublicKey server_b,
   return cfg;
 }
 
+PrioConfig
+PrioConfig_new_int(int num_ints, int prec, PublicKey server_a,
+                   PublicKey server_b, const unsigned char* batch_id,
+                   unsigned int batch_id_len)
+{
+  return PrioConfig_new(num_ints * prec, server_a, server_b, batch_id,
```

Comment:
I think there could be an integer overflow problem here if `num_ints * prec > `INT_MAX`. I think that one way to handle this safely would be to first check that `num_ints <= PrioConfig_maxIntEntries(prec)`.

Reply:
The old PR does ```P_CHECKCB(cfg->precision <= 32); P_CHECKCB(cfg->precision >= 1);```. Do we want to further constrain the precision or just prevent overflows?

Reply:
Either way works for me. I can't think of any realistic applications for using integer values with >32 bits of precision, but it seems fine to allow this in case someone has a reason to do this.